src/modules/remote.json

```diff
@@ -0,0 +1,151 @@
+            "result": {
+              "remoteList": [
+                {
+                  "model": "PR2",
+                  "batteryPercentage": "70"
+                }
+              ]
+            }
+          }
+        ]
+      },
+      {
+        "name": "startPairing",
+        "summary": "Setup the device to allow Remote control to be paired on the next keypress.",
+        "params": [],
+        "result": {
+          "name": "remotePairState",
+          "summary": "The state of currently paired remote",
+          "schema": {
+            "$ref": "#/components/schemas/RemotePairState"
+          }
+        },
+        "examples": [
+          {
+            "name": "Remote State Examples",
+            "params": [],
+            "result": {
+              "pairingState": "pairingRemote"
+            }
+          }
+        ]
+      },
+      {
+        "name": "onRemotePairing",
+        "tags": [
+          {
+            "name": "event"
+          }
+        ],
+        "summary": "Listen for events when the remote control is pairing.",
+        "params": [
+          {
+            "name": "listen",
+            "required": true,
+            "summary": "whether to listen, or stop listening",
+            "schema": {
+              "type": "boolean"
+            }
+          }
+        ],
+        "result": {
+          "name": "remotePairState",
+          "summary": "The state of currently paired remote",
+          "schema": {
+            "$ref": "#/components/schemas/RemotePairState"
+          }
+        },
+        "examples": [
+          {
+              "name": "Default Example",
+              "params": [
+                  {
+                      "name": "listen",
+                      "value": true
+                  }
+              ],
+              "result": {
+                  "name": "Default Result",
+                  "value": {
+                      "event": "Remote.onRemotePairing",
+                      "remotePairState": "pairingComplete"
+                  }
+              }
+          }
+      ] 
+      }
+    ],
+    "components": {
+      "schemas": {
+        "RemoteControlState": {
+          "title": "RemoteList",
+          "type": "array",
+          "description": "Contains a list of remotes paired to the device.",
+          "items": {
+            "$ref": "#/components/schema/RemoteState"
+          }
+        },
+        "RemoteState": {
+          "title": "RemoteState",
+          "type": "object",
+          "description": "The type of keyboard to show to the user",
+          "minProperties": 2,
+          "additionalProperties": false,
+          "required": ["model", "batteryPercentage"],
+          "properties": {
+            "model": {
+              "type": "string",
+              "description": "Model could be PR1, PR2 and other BLE models. It could also be XR11, XR15, XR16 and XR19 for X1 Stbs.",
+              "enum": ["PR1", "PR2", "XR11", "XR15", "XR16", "XR19"]
```

Comment:
Having remote models in the Firebolt spec seems like something we should talk about offline. My instinct is that this enum needs to come from a Distributor configuration, and dan't be part of the Firebolt API spec.

Reply:
Seconded 👍🏻

Reply:
Can the model be just a string